include/spdk/pci_ids.h

```diff
@@ -44,7 +44,9 @@ extern "C" {
 
 #include <stdint.h>
 
+#define SPDK_PCI_ANY_ID			0xffff
 #define SPDK_PCI_VID_INTEL		0x8086
+#define SPDK_PCI_VID_MEMBLAZE	0x1c5f
```

Comment:
Possibly your editor is configured with a different tab size than SPDK uses - our code is formatted with tabs equal to 8 spaces.